content/migration-from-otrs-6.rst

```diff
@@ -286,8 +286,14 @@ Note that the password for the database root is now the password that has been s
 
     docker_admin> docker exec -i otobo_db_1 mysql -u root -p<root_secret> < otrs_schema.sql
     docker_admin> docker exec -i otobo_db_1 mysql -u root -p<root_secret> < otrs_data.sql
-    docker_admin> docker exec -i otobo_db_1 mysql -u root -p<root_secret> -e 'SHOW DATABASES'     # sanity check
-    docker_admin> docker exec -i otobo_db_1 mysql -u root -p<root_secret> otrs -e 'SHOW TABLES'   # sanity check
+
+For a quick checl whether the import worked, you can run the following commands.
```

Comment:
For a quick check whether the import worked, you can run the following commands: